app/Http/Controllers/Randomizer.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class Randomizer extends Controller
```

Comment:
Please add PHPDoc to classes and functions. (/** + Enter, if you are using IDE.)